docs/content/config/best-practices/dkim_dmarc_spf.md

```diff
@@ -148,9 +147,7 @@ DKIM is currently supported by either OpenDKIM or Rspamd:
 
         ---
 
-        In case you have not already provided a default DKIM signing configuration, the script will create one and write it to `/etc/rspamd/override.d/dkim_signing.conf`. If this file already exists, it will not be overwritten.
-
-        When you're already using [the `rspamd/override.d/` directory][docs-rspamd-config-dropin], the file is created inside your volume and therefore persisted correctly. If you are not using `rspamd/override.d/`, you will need to persist the file yourself (otherwise it is lost on container restart).
```

Comment:
This removes a link ref `docs-rspamd-config-dropin` which you'll find at the bottom of the document, could you remove that as well please?